sunpy/tests/figure_tests_env_3.5.yml

```diff
@@ -46,6 +46,7 @@ dependencies:
 - pyparsing=2.1.4=py35_0
 - pyqt=5.6.0=py35_2
 - pytest=3.0.5=py35_0
+- pytest-mock=1.5.0=py35_0
 - python=3.5.2=0
```

Comment:
From Py3.3 mock comes as part of the `unittest.mock` package. Only have to add it
in for the 2.7.yml file